Makefile

```diff
@@ -9,9 +9,10 @@ install:
 	go install
 
 test:
-	go test -v -covermode=count -coverprofile=profile.cov . ./libvirt
-
+	go test -v -covermode=count -coverprofile=profile.cov ./libvirt
+	go test -v .
```

Comment:
this will run the `unit-test` we don\t need for such feature a profile.cov

Reply:
we need to split in 2 otherwise it doesn't work

Reply:
> we need to split in 2 otherwise it doesn't work

What error do you see?  For me:

```console
$ git describe --always --abbrev=40 --dirty
fe5737439750f97f4e1816cf59e90066529f2e72
$ make test
go test -v -covermode=count -coverprofile=profile.cov . ./libvirt
=== RUN   TestPrintVersion
--- PASS: TestPrintVersion (0.32s)
PASS
coverage: 62.2% of statements
ok  	github.com/dmacvicar/terraform-provider-libvirt	0.335s	coverage: 62.2% of statements
...
=== RUN   TestAccLibvirtCloudInit_CreateCloudInitDiskAndUpdate
--- FAIL: TestAccLibvirtCloudInit_CreateCloudInitDiskAndUpdate (0.00s)
panic: interface conversion: interface {} is nil, not *libvirt.Client [recovered]
	panic: interface conversion: interface {} is nil, not *libvirt.Client

goroutine 67 [running]:
testing.tRunner.func1(0xc420460000)
	/home/trking/.local/go/src/testing/testing.go:742 +0x29d
panic(0xdfdea0, 0xc420446180)
	/home/trking/.local/go/src/runtime/panic.go:502 +0x229
github.com/dmacvicar/terraform-provider-libvirt/libvirt.TestAccLibvirtCloudInit_CreateCloudInitDiskAndUpdate(0xc420460000)
	/home/trking/.local/lib/go/src/github.com/dmacvicar/terraform-provider-libvirt/libvirt/resource_libvirt_cloud_init_test.go:24 +0x127b
testing.tRunner(0xc420460000, 0xf5d7b0)
	/home/trking/.local/go/src/testing/testing.go:777 +0xd0
created by testing.(*T).Run
	/home/trking/.local/go/src/testing/testing.go:824 +0x2e0
FAIL	github.com/dmacvicar/terraform-provider-libvirt/libvirt	24.031s
make: *** [test] Error 1
```

But I see that same panic on your 21c6b8b4915b917af0cb080d0e2de14441a59be3 too.

Reply:
Yop the panic is not important. Actually it should not be scheduled that test in the unittest but in acceptance only I still dunno. 

I have an error which complain about the cover profile, which golang version are you using at moment? I think that the thing is maybe changed with higher versions of golang..

Reply:
> I have an error which complain about the cover profile, which golang version are you using at moment? I think that the thing is maybe changed with higher versions of golang..

```console
$ go version
go version go1.10.3 linux/amd64
```